client/cmd/simnet-trade-tests/run

```diff
@@ -86,6 +90,7 @@ ltcelectrumdcr - Litecoin Electrum wallet and Decred RPC wallet on LTC-DCR marke
 dcrdoge - RPC wallets on DCR-DOGE market
 dcrdgb - RPC wallets on DCR-DGB market
 dcreth - Decred RPC wallet and Ethereum native wallet on DCR-ETH market
+dcrfiro - RPC wallets on DCR-FIRO market
```

Comment:
These all passed for me with the miner tmux watches stopped.

Reply:
> These all passed for me with the miner tmux watches stopped.

Me too now .. my issue was an environment issue when running `simnet-trade-tests` binary itself.